app-misc/splitvt/splitvt-1.6.6-r4.ebuild

```
@@ -0,0 +1,53 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+MY_P="${P/-/_}"
+DEB_PL="7"
+
+DESCRIPTION="Splitting terminals into two shells"
+HOMEPAGE="https://slouken.libsdl.org/projects/splitvt/"
+SRC_URI="
+	https://slouken.libsdl.org/projects/splitvt/${P}.tar.gz
+	mirror://debian/pool/main/s/splitvt/${MY_P}-${DEB_PL}.diff.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ia64 ~ppc ~sparc ~x86"
+IUSE=""
+
+DEPEND="sys-libs/ncurses:0="
```

ncurses has only one slot, so we can do...
```suggestion
DEPEND="sys-libs/ncurses:="
```